app/templates/components/footer-main.hbs

```diff
@@ -59,8 +59,8 @@
             </a>
           {{/if}}
           {{#if this.socialLinks.twitterUrl}}
-            <a class="item" href="{{this.socialLinks.twitterUrl}}" target="_blank" rel="noopener noreferrer">
-              <i class="twitter icon"></i> {{t 'Twitter'}}
+            <a class="item" href="https://twitter.com/eventyay" target="_blank" rel="noopener noreferrer">
```

Comment:
you shouldn't change this to hardcoded. it is supposed to be fetch from config

Reply:
also your X is not to scale. refer to other icons and standardisation

Reply:
Hi @cweitat,

Thank you for your feedback. I've made the necessary adjustments to address your concerns:

**Dynamic Configuration:** I've revised the code to ensure that the Twitter logo is fetched from the configuration, as opposed to being hardcoded. This ensures better flexibility and adherence to project standards.

**Icon Scaling:** I've reviewed the scaling of the X icon and adjusted it to align with the standardization of other icons. It should now be more in line with the established design principles.

Please take a moment to review the latest changes in the pull request. Your continued guidance is highly appreciated.

Best regards,
Mohammed Saqlain